libraries/AP_RangeFinder/AP_RangeFinder_LightWareI2C.cpp

```diff
@@ -212,6 +220,14 @@ bool AP_RangeFinder_LightWareI2C::init()
         last_init_time_ms = AP_HAL::millis();
         return true;
     }
+    if (legacy_init()) {
```

Comment:
Any reason for adding this legacy function back?

Reply:
Doh, I am working off of an unupdated mttr-rebase branch. one moment as I fix